Dependency mime failing in windows CI checks

CI checks on the "windows" machine are failing due to the package mime not being able to be installed. 

https://ci.appveyor.com/project/sckott/rsnps/builds/25215851

<summary><strong>From line 438</strong></summary>

```r
/bin/sh: c:/Rtools/mingw_32/bin/gcc: No such file or directory
make: *** [init.o] Error 127
ERROR: compilation failed for package 'mime'
* removing 'c:/RLibrary/mime'
Error in i.p(...) : 
  (converted from warning) installation of package 'mime' had non-zero exit status
Calls: <Anonymous> ... with_rprofile_user -> with_envvar -> force -> force -> i.p
Execution halted
Command exited with code 1
```

How to fix?

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -4,6 +4,9 @@ init:
         Invoke-WebRequest http://raw.github.com/krlmlr/r-appveyor/master/scripts/appveyor-tool.ps1 -OutFile "..\appveyor-tool.ps1"
         Import-Module '..\appveyor-tool.ps1'
 
+environment:
+  USE_RTOOLS: true
+
 install:
   ps: Bootstrap
 

